inputs/elements/working_example2.json

```diff
@@ -0,0 +1,615 @@
+{
+    "economic_data": {
+        "currency": "NOK",
+        "discount_factor": {
+            "unit": "factor",
+            "value": "0.06"
+        },
+        "label": "economic_data",
+        "project_duration": {
+            "unit": "year",
+            "value": "30"
+        },
+        "tax": {
+            "unit": "factor",
+            "value": "0.0 "
+        }
+    },
+    "energyConsumption": {
+        "demand_01": {
+            "dsm": "False",
+            "file_name": "demand_habor.csv",
+            "label": "Habor",
+            "type_asset": "demand",
+            "type_oemof": "sink"
+        },
+        "demand_02": {
+            "dsm": "False",
+            "file_name": "demand_remaining.csv",
+            "label": "Remaining demand",
+            "type_asset": "demand",
+            "type_oemof": "sink"
+        },
+        "demand_03": {
+            "dsm": "False",
+            "file_name": "demand_shore_power.csv",
+            "label": "Shore power (docked ships)",
+            "type_asset": "demand",
+            "type_oemof": "sink"
+        }
+    },
+    "energyConversion": {
+        "diesel_generator": {
```

Comment:
as I see it the "energyVector" we defined was already called "outflow_direction" in the energyConversion.csv. Should I replace it with "energyVector"?

Reply:
energyVector should be an additional parameter here.
A CHP for example would belong both to the electricity/heat sector, and we need to be able to define energyVector individually here. 
-> Please add "energyVector" as a parameter NEXT to outflow/inflow_direction.

Reply:
what's the values of "energyVector" then? Same as on outflow_direction?

Reply:
energyVector may eg. be "Electricity", "Heat", "H2"
Whereas out/inflow_direction may be "Electricity (LES)", "Electricity (PV_01 bus)", "Heat (100 Degree)", "Heat (60 Degree)".

Basically, we will use "energyVector" to group assets appropriatly, eg. for vizualization and cost distribution (in this sense, a CHP is actually quite difficult here as it touches TWO vectors - this will be one of our research questions).